### ISSUE TITLE
[a11y]:  [TableExpandRow] A11y (Accessibility) violation

### ISSUE THREAD

**Issue by loftywaif002** [carbon-design-system/carbon#19082]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.76.0

### React version

18.2.0

### Automated testing tool and ruleset

IBM Equal Accessibility Access checker

### Description

Hi Carbon team,

As part of an IBM-wide requirement, a requirement is for DataStage UI (DataStage/ds-canvas) to be accessibility compliant, via the [IBM Equal Access Accessibility Checker tool](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp?hl=en-US).

### Reproduction/example

https://stackblitz.com/edit/github-voohh6bt

### Steps to reproduce

1. open the stackblitz
2. `run npm install `
3. `npx webpack serve`
4. Click on the top right icon to "`Open Preview in a new tab`"
5. Run the accessibility checker on the page, you should see something like below :

![Image](https://github.com/user-attachments/assets/2771bd3c-e4dd-46f0-9a54-2dfa27e2d042)

Video: 

https://www.dropbox.com/scl/fi/p58d9wf1m65ioi6ugzcy8/Screen-Recording-2025-04-09-at-5.13.07-PM.mov?rlkey=txkg7l22ritunbn2h0i1vnzuz&st=ae3e0fpa&dl=0

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Fixed via https://github.com/carbon-design-system/carbon/pull/20633
